requirements.txt

```diff
@@ -0,0 +1,5 @@
+numpy==1.19.2
+matplotlib==3.3.4
+scikit_learn==0.24.1
+opencv==4.5.1

```

Comment:
I think this should be opencv-python because there is no opencv package to get installed with pip.

Reply:
Yeah, Okay